### ISSUE TITLE
Adopt version 4.20 of the ArcGIS API for JavaScript and enable blend modes and layer effects

### ISSUE THREAD

**Issue by damix911** [visgl/deck.gl#6012]
#### Target Use case
deck.gl should try to load a more recent version of the ArcGIS API for JavaScript, so that more recent features can be used.

Two recent features from which @deck.gl/arcgis users will benefit a lot are [blend modes](https://developers.arcgis.com/javascript/latest/sample-code/intro-blendmode-layer/) and [layer effects](https://developers.arcgis.com/javascript/latest/sample-code/intro-effect-layer/), which are now available in 2D MapViews.

In order to use them, a recent version of the ArcGIS API for JavaScript is needed, and possibly the `DeckLayer` class will need to be modified by adding some requested properties (see the **important** section in the [official docs](https://developers.arcgis.com/javascript/latest/sample-code/custom-gl-visuals/)).

#### Proposed feature
As an example, this is what an app using @deck.gl/arcgis with an Esri basemap and a `TripsLayer` over the `DeckLayer` adapter could look like with the _"bloom"_ effect.

![image](https://user-images.githubusercontent.com/12469086/126879856-926062f3-232b-443e-962e-569fb5c0246e.png)

In code this could be achieved by adding the effect specification to the `DeckLayer`.

```js
  const layer = new DeckLayer({
    'effect': 'bloom(1.5, 0.5px, 0.1)',
    'deck.layers': [
      new TripsLayer(...)
    ]
  });
```

#### To Do List 
  - [ ] Add label and assign to milestone 
  - [ ] Coding
  - [ ] Doc update
  - [ ] What’s new update 
  - [ ] Test

**Comment by Pessimistress** [comment]
Published in v8.5.0

**Comment by MarshallChen** [comment]
Is there a way to have the bloom effects in deck.gl? @Pessimistress
